config/moved_service.go

```diff
@@ -21,6 +21,7 @@ import (
 )
 
 // Service defines the configurable components of the service.
+// Deprecated: Use serviceconfig.Service
```

Comment:
The deprecation message needs a version number

Reply:
Done.